Detectors/ITSMFT/MFT/base/src/HeatExchanger.cxx

```diff
@@ -3738,14 +3738,16 @@ void HeatExchanger::createCoolingPipes(Int_t half, Int_t disk)
   //-----------------------------------------------------------------
   if (disk == 0 || disk == 1 || disk == 2) {
     auto* mCoolingPipe1 = new TGeoVolumeAssembly(Form("cooling_pipe1_H%d", half));
+    auto* mCoolingPipeRear1 = new TGeoVolumeAssembly(Form("cooling_pipeRear1_H%d", half));
+    auto* mCoolingPipeRear2 = new TGeoVolumeAssembly(Form("cooling_pipeRear2_H%d", half));
     if (disk == 0) {
-      length1 = 5.3;
+      length1 = 1.5; //5.3;
```

Comment:
Hi Franck. Thanks.
Is it necessary to keep previous values like this as comments?

Reply:
No sure, I'm going to clean that.